src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -4,8 +4,26 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.SQLFeatureNotSupportedException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.regex.Pattern;
+import org.bson.BsonDocument;
+import org.bson.BsonInt32;
+import org.bson.BsonNull;
+import org.bson.BsonString;
+import org.bson.BsonValue;
 
 public class MongoSQLDatabaseMetaData extends MongoDatabaseMetaData implements DatabaseMetaData {
+    private static final MongoJsonSchema stringSchema;
+    private static final MongoJsonSchema intSchema;
+
+    static {
+        stringSchema = new MongoJsonSchema();
```

Comment:
This is a good idea, I was instantiating these individually in the methods.  I'll switch to using these variables.